packages/indexer-cli/src/commands/indexer/disputes/get.ts

```diff
@@ -24,8 +23,7 @@ module.exports = {
     const { print, parameters } = toolbox
 
     const { h, help, o, output } = parameters.options
-
-
+    const [status, minAllocationClosedEpoch] = parameters.array || []
```

Comment:
I see that `--merged` is part of the help text of this subcommand. I think that needs to be removed.